code/modules/admin/verbs/datumvars.dm

```diff
@@ -927,6 +929,26 @@
 					message_admins("[ADMIN_TPMONTY(usr)] has added [amount] units of [chosen_id] to [A].")
 
 
+	else if(href_list["rotatedatum"])
+		if(!check_rights(R_DEBUG))
+			return
+
+		var/atom/A = locate(href_list["rotatedatum"])
+		if(!istype(A))
+			to_chat(usr, "This can only be done to instances of type /atom")
+			return
+
+		switch(href_list["rotatedir"])
+			if("right")
+				A.setDir(turn(A.dir, -45))
```

Comment:
some things won't play nicely with 45 degree rotations

Reply:
This is /tg/ code, what would you suggest?